backend/api/analysis/sentiment.py

```diff
@@ -80,43 +74,30 @@ def sentiment_analysis_example(client, documents):
 
     return response
 
-### Input: headers (list), kind_of_search (str)
-### Performs a sentimental analysis utilizing the Twitter/Azure API and returns a json structured response 
-def analyze(headers, kind_of_search):
 
-    ### Authenticate on Twitter and return if username is invalid or empty tweets 
+def analyze(headers, kind_of_search):
     url = create_twitter_url_req([headers[0], headers[1]], kind_of_search)
+    # bearer_token = process_yaml("search_tweets_api")
     res_json = twitter_auth_and_connect(process_env("search_tweets_api"), url)
-    
```

Comment:
Current sentiment.py file seems to not have the updated comment/error checks that's currently being deployed at the moment.

Reply:
That makes sense, though – this is the frontend branch. It's behind the main/backend branch by definition, and I won't be overwriting any of the main/backend commits with backend files from the frontend because that wouldn't make any sense